File: project-domain/Event.cs
using System;
using System.Collections.Generic;
namespace project.domain
{
    public class Event
    {
        public int Id { get; set; }
        public string Place { get; set; }
        public DateTime Date { get; set; }
        public string Theme { get; set; }
        public int AmountPeople { get; set; }
        public string ImagemUrl { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public List<Lot> Lots { get; set; }
        public List<SocialNetwork> SocialNetworks { get; set; }
        public List<SpeakerEvent> SpeakerEvents { get; set; }
    }
}

File: project-web/src/app/models/event.ts
import { iLot } from "./lot";
import { iSocialNetwork } from "./social-network";
import { iSpeaker } from "./speaker";

export class iEvent {
  constructor() {}
  id: number;
  date: string;
  place: string;
  theme: string;
  amountPeople: number;
  imagemUrl: string;
  phone: string;
  email: string;
  lots: iLot[];
  socialNetworks: iSocialNetwork[];
  speakerEvents: iSpeaker[];
}
